### PR TITLE
Removed tips and tricks

### PR DESCRIPTION
**Pull request by Andrii-Yezenitskyi**
Task [#4218](https://github.com/ita-social-projects/GreenCity/issues/4218).

### CODE REVIEW

**Review by Max877866 — APPROVED** [review]

**Review by NataliaHlazova — APPROVED** [review]

**Review by Pavl0-Gural — APPROVED** [review]

File: dao/src/main/resources/db/changelog/db.changelog-master.xml [hunk 1]
```diff
@@ -34,25 +34,20 @@
...
     <include file="db/changelog/logs/ch-habit-status-Beshlei-2.xml"/>
     <include file="db/changelog/logs/ch-habits-Beshlei-1.xml"/>
     <include file="db/changelog/logs/ch-habit-status-calendar-Beshlei-1.xml"/>
     <include file="db/changelog/logs/ch-primarykey-eco-news-tags-Lehkyi-1.xml"/>
-    <include file="db/changelog/logs/ch-primarykey-tips-and-tricks-tags-Lehkyi.xml"/>
     <include file="db/changelog/logs/ch-fact-of-the-day-Lehkyi.xml"/>
     <include file="db/changelog/logs/ch-social-network-Lehkyi.xml"/>
     <include file="db/changelog/logs/ch_rating_statictics_Dovganyuk.xml"/>
     <include file="/db/changelog/logs/ch-user-username-column-type.xml"/>
-    <include file="/db/changelog/logs/ch-tips-and-tricks-translations-Petryshak.xml"/>
```
Comment (VadymMakitra) [inline dao/src/main/resources/db/changelog/db.changelog-master.xml]: Don't remove the change set, just write new, that will delete the table or column that you need to delete

File: dao/src/main/resources/db/changelog/logs/ch-insert-into-tag-translations-Kozak.xml [hunk 2]
```diff
@@ -93,112 +93,22 @@
...
-            <column name="name">Eco-city</column>
-            <column name="tag_id">10</column>
-            <column name="language_id">2</column>
-        </insert>
-        <insert tableName="tag_translations">
-            <column name="id">30</column>
-            <column name="name">Эко-город</column>
-            <column name="tag_id">10</column>
-            <column name="language_id">3</column>
-        </insert>
```
Comment (VadymMakitra) [inline dao/src/main/resources/db/changelog/logs/ch-insert-into-tag-translations-Kozak.xml]: Don't remove the change set, just write new, that will delete the table or column that you need to delete

**Review by VadymMakitra — CHANGES_REQUESTED** [review]
Resolve comment's

File: dao/src/main/resources/db/changelog/db.changelog-master.xml (hunk 1 above)
Comment (thesvider) [reply dao/src/main/resources/db/changelog/db.changelog-master.xml]: agree. create separate changeset that will cleanup unused tables/columns

Comment (Andrii-Yezenitskyi) [reply dao/src/main/resources/db/changelog/db.changelog-master.xml]: Done

File: dao/src/main/resources/db/changelog/logs/ch-insert-into-tag-translations-Kozak.xml (hunk 2 above)
Comment (Andrii-Yezenitskyi) [reply dao/src/main/resources/db/changelog/logs/ch-insert-into-tag-translations-Kozak.xml]: Done

**Review by VadymMakitra — APPROVED** [review]

**Pull request merged**